components/dashboard/src/start/VerifyModal.tsx

```diff
@@ -0,0 +1,201 @@
+/**
+ * Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License-AGPL.txt in the project root for license information.
+ */
+
+import { useState } from "react";
+import Alert, { AlertType } from "../components/Alert";
+import Modal from "../components/Modal";
+import { getGitpodService } from "../service/service";
+import PhoneInput from "react-intl-tel-input";
```

Comment:
**question**: Unless there's a strong preference over this library, given the styling issues of the input, shall we consider switching to a different library like [`intl-tel-input`](https://github.com/jackocnr/intl-tel-input) which also happens to be sponsored by Twilio? Seems more popular and flexible, no?

> 270 vs. 6.4K ⭐

Reply:
It is the react version of that library.